src/util/ViewAnimation.js

```diff
@@ -100,10 +100,10 @@ const NoAnimation = {
    * Zoom to the given location by not using animations.
    * @param {ol.View} view The `ol.View` of the map.
    * @param {ol.Coordinate} location The destination center point to zoom to.
-   * @param {function(complete)} completionCallback An optional notification that the animation has completed.
+   * @param {function(complete)} [completionCallback] An optional notification that the animation has completed.
    * @param {Object} options Configuration object for the animation, supported attributes are:
-   * * {Number} zoom An optional final zoom level.
-   * * {Number} maxZoom An optional maximal zoom level.
+   * * {Number} [zoom] An optional final zoom level.
+   * * {Number} [maxZoom] An optional maximal zoom level.
```

Comment:
```suggestion
   * @param {Number} [options.zoom] An optional final zoom level.
   * @param {Number} [options.maxZoom] An optional maximal zoom level.
```

To conform to official [JSDoc Parameters with properties](https://jsdoc.app/tags-param)